modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +55,343 @@ docker compose version
+// (step {"action":"find", "selector": "[data-testid='auth-submit']", "click": true})
+// (step {"action":"wait", "duration": 15000})
+// (step {"action":"saveScreenshot", "path": "overview.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+To view details about a specific broker, click *View* at the end of the row in the *Broker Details* table.
+
+// (step {"action":"find", "selector": ".chakra-table tbody tr:last-child td:last-child button", "matchText": "View", "click": true})
+// (step {"action":"wait", "duration": 15000})
+// (step {"action":"saveScreenshot", "path": "broker-overview.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+image::console:broker-overview.png[]
+
+See also: xref:console:config/security/plain.adoc[].
+
+=== View topics and filter messages
+
+This deployment comes with two pre-configured topics, `logins` and `transactions`, which are actively receiving data as part of a data pipeline deployed through Redpanda Connect. These topics simulate real-world data flows, enabling you to explore and interact with Redpanda Console features.
+
+You can filter messages in topics using several methods:
+
+- Text search: Search for specific text within the message.
+- Partition offset filters: Set the start and end offsets to narrow down the message range.
+- JavaScript filters: Use JavaScript function bodies to create complex filters that run server-side.
+
+Suppose you're asked to find all transactions related to the `.edu` domain. You can use a JavaScript filter to display only messages that include email addresses in that domain.
+
+. In the left menu, click *Topics*.
+// (step {"action":"find", "selector": "[data-testid='nav-link-Topics']", "matchText": "Topics", "click": true})
+. Click the *transactions* topic. When the topic's page opens, the *Messages* tab is selected by default.
+// (step {"action":"goTo", "url": "http://localhost:8080/topics/transactions"})
+// (step {"action":"wait"})
+. Click *Add filter* > *JavaScript Filter*.
+// (step {"action":"saveScreenshot", "path": "topic.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+// (step {"action":"find", "selector": "[data-testid='add-topic-filter']", "matchText": "Add filter", "click": true, "timeout": 10000})
+// (step {"action":"find", "selector": "[data-testid='add-topic-filter-javascript']", "matchText": "JavaScript Filter", "click": true, "timeout": 10000})
+. Give your filter a name, such as "Find .edu domains".
+// (step {"action":"find", "selector": "[data-testid='add-javascript-filter-name']", "typeKeys": "Find .edu domains", "click": true, "timeout": 10000})
+// (step {"action":"saveScreenshot", "path": "js-filter.png", "directory": "../../modules/console/images", "overwrite": "byVariation", "maxVariation": 10})
++
+image::console:js-filter.png[]
```

Comment:
It might be more helpful to put the screenshot after the next step, including the code in the screenshot